spec/services/atmosphere/cloud/vmt_updater_spec.rb

```diff
@@ -56,6 +56,24 @@
     expect(target_vmt.version).to eq 13
   end
 
+  it 'updates old VMT version and relation to AT when metadata appears' do
+    at, source_vmt, source_cs = at_with_vmt_on_cs
```

Comment:
Useless assignment to variable - `at`. Use `_` or `_at` as a variable name to indicate that it won't be used.